odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/handlers/PartnerHandler.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.handlers;
+
+import static java.util.Arrays.asList;
+
+import com.ozonehis.eip.odoo.openmrs.Constants;
+import com.ozonehis.eip.odoo.openmrs.client.OdooClient;
+import com.ozonehis.eip.odoo.openmrs.client.OdooUtils;
+import com.ozonehis.eip.odoo.openmrs.mapper.odoo.PartnerMapper;
+import com.ozonehis.eip.odoo.openmrs.model.Partner;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.ProducerTemplate;
+import org.hl7.fhir.r4.model.Patient;
+import org.openmrs.eip.EIPException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Setter
+@Component
+public class PartnerHandler {
+
+    @Autowired
+    private OdooClient odooClient;
+
+    @Autowired
+    private PartnerMapper partnerMapper;
+
+    public Partner partnerExists(String partnerRefID) {
```

Comment:
Rename this to something like getPartner

Reply:
Renamed to `getPartnerByID`